.github/workflows/test-rc-release.yaml

```diff
@@ -37,9 +104,14 @@ jobs:
           exit 1
 
   create-branch-for-testing-rc-release:
-    needs: validate-manual-input
+    needs: [validate-manual-input, check-rc-testing-annocement]
```

Comment:
This needs to be an `or` condition right? validate-manual-input runs only on `workflow_dispatch` whereas `check-rc-testing-annocement` runs only on `schedule`, so both will not be true at the same time and this job will never be executed then, no?

Reply:
AFAIK, we don't have a way to do it in `needs`. That's why I use `always()` [here](https://github.com/astronomer/astronomer-providers/pull/1146/files#diff-7cd74bfb9babddfbfa7911a72e5d0784f450360d0a81e9f8319ecba30711b059R110)

Reply:
okay, so just for my understanding, we are guaranteed that this job will run, right? Although `validate-manual-input` or `check-rc-testing-annocement` is skipped based on their if conditions, since we have always, this job will be run, correct? If that is correct, yes, please mark this conversation as resolved. Sorry for my lack of expertise here :)

Reply:
Yes, that's how it works. Thanks for looking so deep into it 🙌 It also took me some time to sort all these stuff out